Mathlib/LinearAlgebra/Dual.lean

```diff
@@ -1599,6 +1599,12 @@ theorem dualMap_bijective_iff {f : V₁ →ₗ[K] V₂} :
   simp_rw [Function.Bijective, dualMap_surjective_iff, dualMap_injective_iff, and_comm]
 #align linear_map.dual_map_bijective_iff LinearMap.dualMap_bijective_iff
 
+lemma dualAnnihilator_ker_eq_range_of_comm (f : V₂ →ₗ[K] Dual K V₂) (h : ∀ x y, f x y = f y x) :
```

Comment:
```suggestion
lemma dualAnnihilator_ker_eq_range_of_comm (f : V₂ →ₗ[K] Dual K V₂) (h : f.flip = f) :
```
is generally easier to prove things about (since `ext` can apply)